packages/tizen_bundle/example/README.md

```diff
@@ -0,0 +1,16 @@
+# tizen_bundle_example
+
+Demonstrates how to use the tizen_bundle plugin.
+
+## Getting Started
```

Comment:
Please remove this section (the auto-generated lines).

Reply:
Done.